src/workloads/scale/MultiPlanStorm.yml

```diff
@@ -0,0 +1,93 @@
+SchemaVersion: 2018-07-01
```

Comment:
I'm leaving this comment for the file name: `MultiPlanStorm` might not be specific enough. In this case we want to generate a variant of a multi-plan storm which results in unbounded growth of the deduplicated set of RecordIds during an index scan. Can we encode that description into the test name, as well as improve the `Description` field below?

Reply:
Modified description and renamed.